test/common/http/async_client_impl_test.cc

```diff
@@ -314,7 +314,7 @@ TEST_F(AsyncClientImplTest, BasicHashPolicy) {
           Invoke([&](Upstream::ResourcePriority, auto,
                      Upstream::LoadBalancerContext* context) -> Http::ConnectionPool::Instance* {
             // this is the hash of :path header value "/"
-            EXPECT_EQ(16761507700594825962UL, context->computeHashKey().value());
+            EXPECT_NE(0, context->computeHashKey().value());
```

Comment:
sorry. this was my mistake. Can you restore the original compare vs constant, and add a comment that we want to ensure stability of the hash values across releases?

Reply:
no worries, I appreciate your review, that improve my c++ skill :)